pkg/app/up.go

```diff
@@ -216,6 +216,12 @@ func up(clicontext *cli.Context) error {
 		numGPU = 1
 	}
 
+	shmSize := clicontext.Int("shm-size")
+	isSetShmSize := clicontext.IsSet("shm-size")
+	if !isSetShmSize {
+		shmSize = builder.ShmSize()
+	}
```

Comment:
This argument has a default value of 2048 (MB). What if the builder doesn't have this value?